libpod/oci.go

```diff
@@ -282,6 +284,22 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string) (er
 		"args": args,
 	}).Debugf("running conmon: %s", r.conmonPath)
 
+	if selinux.GetEnabled() {
+		// Set the label of the conmon process to be level :s0
+		// This will allow the container processes to talk to fifo-files
+		// passed into the container by conmon
+		plabel, err := selinux.CurrentLabel()
+		if err != nil {
+			return errors.Wrapf(err, "Failed to get current SELinux label")
+		}
+
+		c := selinux.NewContext(plabel)
+		if c["level"] != "s0" && c["level"] != "" {
```

Comment:
How would level be set to "" and wouldn't we want to set it to 's0' anyway?  This is probably a question exposing my selinux lack of knowledge.

Reply:
> How would level be set to ""...

That's for [systems that don't support levels][1].

[1]: https://github.com/projectatomic/libpod/pull/854#discussion_r191910714

Reply:
Ah, that's makes sense.  Thanks for the ptr @wking